bay-area/config.yml

```diff
@@ -53,10 +53,12 @@ graphhopper:
       vehicle: bike2
       weighting: fastest
       turn_costs: true
-      u_turn_costs: 10
+      u_turn_costs: 
```

Comment:
This looks like a possible merging/rebasing mistake, unless this is meant to be zeroed out?